st2common/st2common/config.py

```diff
@@ -567,6 +567,48 @@ def register_opts(ignore_errors=False):
 
     do_register_opts(metrics_opts, group='metrics', ignore_errors=ignore_errors)
 
+    # Common timers engine options
+    timer_logging_opts = [
+        cfg.StrOpt(
+            'logging', default=None,
+            help='Location of the logging configuration file. '
+                 'NOTE: Deprecated in favor of timersengine.logging'),
+    ]
+
+    timers_engine_logging_opts = [
+        cfg.StrOpt(
+            'logging', default='/etc/st2/logging.timersengine.conf',
+            help='Location of the logging configuration file.')
+    ]
+
+    do_register_opts(timer_logging_opts, group='timer', ignore_errors=ignore_errors)
+    do_register_opts(timers_engine_logging_opts, group='timersengine', ignore_errors=ignore_errors)
+
+    # NOTE: We default old style deprecated "timer" options to None so our code
+    # works correclty and "timersengine" has precedence over "timers"
+    # NOTE: "timer" section will be removed in v3.1
+    timer_opts = [
+        cfg.StrOpt(
+            'local_timezone', default=None,
```

Comment:
I read the original code and default=True for both timers and timersengine was pretty dumb on my part. I now get what you mean. Good catch.